doc/reference/modules/query_linq.xml

```diff
@@ -67,6 +67,10 @@ using NHibernate.Linq;]]></programlisting>
     session.Query<Cat>()
         .Where(c => c.BirthDate == DateTime.Today.MappedAs(NHibernateUtil.Date))
         .ToList();]]></programlisting>
+    <programlisting><![CDATA[IList<Cat> cats =
+    session.Query<Cat>()
+        .Where(c => c.Name == "Max".MappedAs(TypeFactory.Basic("AnsiString(200)")))
+        .ToList();]]></programlisting>
```

Comment:
Sneakily added another example here by the way... Not utterly unrelated, since `MappedAs` benefits from `DBOnly` (avoids having its implementation relying on throwing exceptions), but a bit remote maybe.